Common/Enums/VehicleRole.sqx

```diff
@@ -15,7 +15,8 @@ namespace Intrusion.Common
 		Driver = 1,
 		Gunner = 2,
 		Cargo = 3,
-		Commander = 4
+		Commander = 4,
+		Turret = 5
```

Comment:
Not sure if this is correct. A person in a turret, does it not always have the role of a gunner? Let's not confuse the vehicle role with vehicle position. Do we need a new VehiclePosition enum?